extractor/src/main/java/org/schabi/newpipe/extractor/services/bandcamp/extractors/BandcampChannelExtractor.java

```diff
@@ -55,12 +60,20 @@ public String getBannerUrl() throws ParsingException {
                             .get(replaceHttpWithHttps(channelInfo.getString("bandcamp_url")))
                             .responseBody();
 
-            return Stream.of(Jsoup.parse(html).getElementById("customHeader"))
+            final List<Image> imageList = new ArrayList<>();
+
+            Stream.of(Jsoup.parse(html).getElementById("customHeader"))
                     .filter(Objects::nonNull)
                     .flatMap(element -> element.getElementsByTag("img").stream())
-                    .map(element -> element.attr("src"))
-                    .findFirst()
-                    .orElse(""); // no banner available
+                    .forEach(element -> {
+                        final String url = element.attr("src");
+                        if (!isNullOrEmpty(url)) {
+                            imageList.add(new Image(replaceHttpWithHttps(url),
+                                    HEIGHT_UNKNOWN, WIDTH_UNKNOWN));
+                        }
+                    });
+
+            return Collections.unmodifiableList(imageList);
```

Comment:
```suggestion
          return Stream.of(Jsoup.parse(html).getElementById("customHeader"))
                    .filter(Objects::nonNull)
                    .flatMap(element -> element.getElementsByTag("img").stream())
                    .map(element -> element.attr("src"))
                    .filter(url -> !isNullOrEmpty(url))
                    .map(url -> new Image(replaceHttpWithHttps(url), HEIGHT_UNKNOWN, WIDTH_UNKNOWN))
                    .collect(Collectors.collectingAndThen(Collectors.toList(), Collections::unmodifiableList));
```

Reply:
Done, I also applied your suggestion in other places I found it was relevant to do the changes.